plugins/action/k8s_info.py

```diff
@@ -71,6 +71,42 @@ def run(self, tmp=None, task_vars=None):
         if src:
             new_module_args['src'] = src
 
+        template = self._task.args.get('template', None)
```

Comment:
The new param needs docs—and I would suggest also making sure to put something along the lines of "the template must be local to the machine running the Ansible playbook" or something like that. I know one of the original objections to template support was that you couldn't use templates on a remote source.